lib/cryptoexchange/exchanges/bitcoin_indonesia/bitcoin_indonesia.yml

```diff
@@ -0,0 +1,19 @@
+:pairs:
```

Comment:
I want to access the keys using `:`, any better ideas? Write custom methods? The helper methods are only available in Rails package i think